pkg/admission/provider.go

```diff
@@ -1,93 +1,91 @@
 // TODO: only supports plain text with no auth, add security later
-// TODO: use keystone to access Neutron API
+// TODO: check keystone expiration date 
 package admission
 
 import (
-	resty "gopkg.in/resty.v1"
+	"fmt"
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/openstack"
+	"github.com/gophercloud/gophercloud/openstack/networking/v2/networks"
+	"github.com/gophercloud/gophercloud/openstack/networking/v2/ports"
+)
+
+const (
+	providerDefaultRegion  = "RegionOne"
```

Comment:
This value should not be hardcoded.
What happens if you remove this option?
If you cannot remove this constant, it has to be configurable like username and password.

Reply:
After some investigation it seems like it can be removed.